packages/cdktf/lib/testing/matchers.ts

```diff
@@ -115,35 +129,33 @@ function getAssertElementWithProperties(
     const pass = passEvaluation(items, properties);
 
     if (pass) {
-      return {
-        pass,
-        message: () =>
-          `Expected no ${
-            itemType.tfResourceType
-          } with properties ${JSON.stringify(
-            properties,
-            jestAsymetricMatcherStringifyReplacer
-          )} to be present in synthesised stack.
+      return new AssertionReturn(
+        `Expected no ${
```

Comment:
@DanielMSchmidt this was here before, but why is it "expected **no**"? Shouldn't it be "expected ...  to be present"?

Reply:
Or is this for the behavior of Jest? as `pass` is true, Jest will use this error message in case a user chained `.not.` when using the matcher?

Reply:
Yeah, exactly that is what is happening under the hood. Jest allows you to have specific error messages for the inverse version. You basically return a message in case the result is not the expected one. So in case true is not the expected one a user must have used not (otherwise true would be expected).